### PR TITLE
RegExp in filter with mongo to filter users with lowercase in names

### PR DESCRIPTION
**Pull request by ealexisaraujo**

### CODE REVIEW

File: components/users/usersService.js [hunk 1]
```diff
@@ -41,7 +41,9 @@ class usersService {
       if (regExpId.test(args[criteria])) {
         return { [criteria]: parseInt(args[criteria]) };
       } else {
-        return { [criteria]: args[criteria] };
+        return {
+          [criteria]: { $regex: new RegExp(`^${args[criteria]}`, 'i') },
```
Comment (hyfi06) [inline components/users/usersService.js:45]:
```js
new RegExp(`.*${args[criteria].*}`, 'i') }
```
 for substrings

Comment (hyfi06) [reply components/users/usersService.js:45]: Test RegExp with criteria numbers for reduce all to one case

Comment (ealexisaraujo) [reply components/users/usersService.js:45]: Okay! Thanks @hyfi06 for you advices!

Comment (ealexisaraujo) [reply components/users/usersService.js:45]: I changed for this:

new RegExp(`.*${args[criteria]}.*`, 'i') }

Regards!

**Pull request merged**